api/candlepin-api-spec.yaml

```diff
@@ -509,6 +509,166 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /consumers/{consumer_uuid}/content_overrides:
+    get:
+      description: Retrieves list of Content Overrides
+      tags:
+        - Consumer
+        - Content Override
+      operationId: listConsumerContentOverrides
+      security: []
+      parameters:
+        - name: consumer_uuid
+          in: path
+          description: The ID of the consumer
+          required: true
+          schema:
+            type: string
+      x-java-response:
+        type: Iterable
+        isContainer: true
+      responses:
+        200:
+          description: List of content overrides of the Consumer
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ContentOverrideDTO'
+        400:
```

Comment:
One minor change here as I think it should be ```404``` originating from ```verifyAndLookupConsumer``` method of consumer curator class.

Reply:
Done.